templates/PlayFab_InstanceApi.cpp.ejs

```diff
@@ -95,14 +95,14 @@ namespace PlayFab
     {
 <%- getRequestActions("        ", apiCall, true) %>
         IPlayFabHttpPlugin& http = *PlayFabPluginManager::GetPlugin<IPlayFabHttpPlugin>(PlayFabPluginContract::PlayFab_Transport);
-        const auto requestJson = request.ToJson();
+        const auto& requestJson = request.ToJson();
```

Comment:
similar concerns as elsewhere

explicit type here

auto instead of unique_ptr on 105

explicit type on 130 and 147